src/ToolBox/SOS/Strike/util.cpp

```diff
@@ -6284,6 +6285,11 @@ HRESULT SymbolReader::LoadCoreCLR() {
                         SymbolReaderClassName, "LoadSymbolsForModule",
                         (void **)&loadSymbolsForModuleDelegate);
     IfFailRet(Status);
+    Status = CreateDelegate(hostHandle, domainId, SymbolReaderDllName,
+                        SymbolReaderClassName, "GetLocalVariableName",
+                        (void **)&getLocalVariableNameDelegate);
+    IfFailRet(Status);
```

Comment:
BTW, it looks like in this change and in some previous changes, the following pattern has been added:

```c++
Status = Something();
IfFailRet(Status);
```

whereas the convention is normally:

```c++
IfFailRet(Something());
```